src/libraries/System.Security.Cryptography.Xml/src/System.Security.Cryptography.Xml.csproj

```diff
@@ -4,6 +4,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Nullable>disable</Nullable>
     <NoWarn>$(NoWarn);CA1850</NoWarn> <!-- CA1850 suppressed due to multitargeting -->
+    <Nullable>enable</Nullable>
```

Comment:
```suggestion
```

Reply:
Removed. I also just noticed that there were **two** `<Nullable>` elements - probably a slip up when rebasing (there's been a few on this PR!)